datalake/src/main/java/com/sequenceiq/datalake/service/sdx/SdxService.java

```diff
@@ -171,6 +176,29 @@ public SdxCluster createSdx(final String userCrn, final String name, final SdxCl
         return sdxCluster;
     }
 
+    private void validateCloudStorageRequest(SdxCloudStorageRequest cloudStorage) {
+        if (cloudStorage != null) {
+            ValidationResultBuilder validationBuilder = new ValidationResultBuilder();
```

Comment:
can we also check that the baselocation contains the protocol ?  @attilapalfi92

Reply:
For the telemetry (LOG identity) it's not necessary. I'm checking if it's necessary for SDX (IDBroker identity)

Reply:
cool thanks

Reply:
added the validation for SDX